website/docs/d/stack_hci_storage_path.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_stack_hci_storage_path"
+description: |-
+  Gets information about an existing Stack HCI Storage Path.
+---
+
+# Data Source: azurerm_stack_hci_storage_path
+
+Use this data source to access information about an existing Stack HCI Storage Path.
+
+## Example Usage
+
+```hcl
+data "azurerm_stack_hci_storage_path" "example" {
+  name                = "existing"
+  resource_group_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_stack_hci_storage_path.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Stack HCI Storage Path.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Stack HCI Storage Path exists. Changing this forces a new Stack HCI Storage Path to be created.
```

Comment:
We're not recreating a datasource so that line can be removed

```suggestion
* `resource_group_name` - (Required) The name of the Resource Group where the Stack HCI Storage Path exists. 
```

Reply:
fixed